src/PushwooshMessage.php

```diff
@@ -248,4 +252,23 @@ public function useRecipientTimezone()
 
         return $this;
     }
+
+    /**
+     * Enable or disable silent notifications for both Android and iOS platforms.
+     *
+     * @param bool $silent
+     * @return $this
+     */
+    public function silent(bool $silent = false)
```

Comment:
I think `true` is a more sensible default. Any particular reason you changed it to `false`?
```suggestion
    public function silent(bool $silent = true)
```

Reply:
Yes, that's right